package.json

```diff
@@ -7,8 +7,8 @@
     "build": "pnpm -r build",
     "start": "pnpm --filter @graphprotocol/docs start",
     "export": "pnpm --filter @graphprotocol/docs export",
-    "lint": "pnpm typecheck && eslint . --ext .js,.jsx,.ts,.tsx,.mjs --cache --max-warnings 0 && pnpm prettier:check",
-    "lint:fix": "pnpm typecheck; eslint . --ext .js,.jsx,.ts,.tsx,.mjs --fix; pnpm prettier; exit 0",
+    "lint": "eslint --cache --ext .js,.jsx,.ts,.tsx,.mjs --max-warnings 0 . && pnpm prettier:check && pnpm typecheck",
+    "lint:fix": "eslint --cache --ext .js,.jsx,.ts,.tsx,.mjs --fix . && pnpm prettier && pnpm typecheck",
```

Comment:
Oh, why revert this?

Reply:
sorry, mistake from rebase, now replaced only `;` by `&&` as I mentioned https://github.com/graphprotocol/docs/pull/319#discussion_r1140659507